tests/hpc/enable_in_zypper.pm

```diff
@@ -29,7 +29,10 @@ sub run {
     assert_script_run "zypper lr | grep $reponame";
 
     zypper_call("--gpg-auto-import-keys ref");
-    zypper_call 'up';
+
+    # The source where we get ganglia from at the moment cannot be updated
+    # needs to be changed once ganglia is officially released
+    zypper_call 'up' if !check_var('HPC', 'ganglia-server') || !check_var('HPC', 'ganglia-client');
```

Comment:
if zypper up can ruin test , I would record soft failure here.

Reply:
but with what reference? we should think about this and come up with a better solution